test/e2e-tests-ingress.sh

```diff
@@ -34,7 +34,8 @@ function wait_until_taskrun_completed() {
     case ${reason} in
     "TaskRunValidationFailed"|"TaskRunTimeout")
       echo -e "\n\nERROR: ${reason}"
-      kubectl get taskrun $1 -o=jsonpath='{.status.conditions[0].message}'
+      # kubectl get taskrun $1 -o=jsonpath='{.status.conditions[0].message}'
+      kubectl get taskrun $1 -o yaml
```

Comment:
Was this intentional?

Reply:
There's a flaky failing test in the taskrun setup, I've dropped this commit, but the tests are failing integration tests are failing intermittently.

Reply:
And yes, a logger config per event listener does sound useful.

The code also fixes a typo in the logging file `"\"developmen\": false,"`

Reply:
Interesting. Opened https://github.com/tektoncd/triggers/issues/254 to investigate